docs/zh-cn/api_docs.md

```diff
@@ -0,0 +1,68 @@
+# QMK API
+
+本章节详述了QMK API的使用方法，若您是应用开发者，使用这套API可以实现[QMK](https://qmk.fm)键盘固件的编译支持。
```

Comment:
```suggestion
本章节详述了QMK API的使用方法，若您是应用开发者，使用这套 API 可以实现 [QMK](https://qmk.fm) 键盘固件的编译支持。
```

Reply:
For copywriting, please refer to https://github.com/sparanoid/chinese-copywriting-guidelines.

Reply:
for uppercase words, splitting with spaces looks void and not fluent.I may open a issue later talking about this.